.github/actions/repo-gardening/readme.md

```diff
@@ -0,0 +1,8 @@
+# GitHub repository gardening
```

Comment:
Do you think it would be useful to document how everything is working here? I was a bit confused by looking into the`gardening.yml` file. I assume this is a custom JS action, that we define here.

Reply:
I've added some extra content in e26d312c62eb431ae802cc328482466cf6330f99. Does it make a bit more sense? I'm happy to go into more details if needed. It can definitely be a bit confusing at first since we have an action within an action.

Let me know if you need me to clarify anything else, I'll be happy to!